.azure/azure-pipelines.perf.yml

```diff
@@ -194,9 +194,9 @@ stages:
         extraName: 'xdp'
         extraPrepareArgs: -DisableTest -InstallXdpSdk
         ${{ if eq(parameters.pgo_mode, false) }}:
-          extraBuildArgs: -DisableTest -DisableTools -UseXdp -ExtraArtifactDir Xdp
+          extraBuildArgs: -DisableTest -DisableTools -ExtraArtifactDir Xdp
         ${{ if eq(parameters.pgo_mode, true) }}:
-          extraBuildArgs: -DisableTest -DisableTools -UseXdp -ExtraArtifactDir Xdp -PGO
+          extraBuildArgs: -DisableTest -DisableTools -ExtraArtifactDir Xdp -PGO
```

Comment:
Remove this XDP specific build all together, and have the XDP run use the normal build (but with XDP installed).